spec.bs

```diff
@@ -1096,6 +1097,14 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
     * To handle an error:
       1. Set |auctionConfig|'s [=auction config/direct from seller signals header ad slot=] to
         failure.
+1. If |config|["{{AuctionAdConfig/deprecatedRenderURLReplacements}}"] [=map/exists=]:

```

Comment:
Need to check if there are component auctions and throw if there are.
https://crsrc.org/c/third_party/blink/renderer/modules/ad_auction/navigator_auction.cc;drc=738f1250512a5e6ff8cacb6b36540cd50e8156d8;l=2490.

You can do it [here](https://github.com/WICG/turtledove/blob/main/spec.bs#L1203).

Reply:
Done.